src/System.IO.FileSystem.Watcher/tests/project.json

```diff
@@ -24,13 +24,36 @@
     "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00807-03"
   },
   "frameworks": {
-    "netstandard1.3": {}
+    "netstandard1.3": {},
+    "netstandard1.7": {
+      "dependencies": {
+        "System.Runtime.Serialization.Formatters": "4.3.0-beta-24522-03"
+      }
+    }
   },
   "supports": {
     "coreFx.Test.netcoreapp1.0": {},
     "coreFx.Test.net46": {},
     "coreFx.Test.net461": {},
     "coreFx.Test.net462": {},
-    "coreFx.Test.net463": {}
+    "coreFx.Test.net463": {},
+    "coreFx.Test.netcoreapp1.1-ns17": {
+      "netstandard1.7": [
+        "win7-x86",
+        "win7-x64",
+        "win10-arm64",
+        "osx.10.10-x64",
+        "centos.7-x64",
+        "debian.8-x64",
+        "rhel.7-x64",
+        "ubuntu.14.04-x64",
+        "ubuntu.16.04-x64",
+        "ubuntu.16.10-x64",
+        "fedora.23-x64",
+        "linux-x64",
+        "opensuse.13.2-x64",
+        "opensuse.42.1-x64"
+      ]
+    }
```

Comment:
I believe you shouldn't need this anymore. All you need now is to add a `"coreFx.Test.netcoreapp1.1": {},` and that should be good enough since we recently updated the nuget framework mappings which understand the connection between netstandard1.7 and netcoreapp1.1